examples/src/main/scala/com/github/gvolpe/tracer/ZIOServer.scala

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2018 com.github.gvolpe
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.github.gvolpe.tracer
+
+import com.github.gvolpe.tracer.instances.tracer._
+import com.github.gvolpe.tracer.instances.tracerlog._
+import scalaz.zio.{App, Clock, IO}
+import scalaz.zio.interop.Task
+import scalaz.zio.interop.catz._
+
+object ZIOServer extends App {
+
+  implicit val clock = Clock.Live
+  implicit val timer = ioTimer[Throwable]
```

Comment:
I think this line is not needed, only a clock (`ioTimer` is implicit, in scope and will find the clock it needs).